releases/release-1.24/exceptions.yaml

```diff
@@ -0,0 +1,22 @@
+# Exception requests in 1.24
+# Google Group: https://groups.google.com/forum/#!forum/kubernetes-sig-release
+# Release Team Lead: James Laverack ([@JamesLaverack](https://github.com/JamesLaverack) 
+
+# Enhancements Freeze Exceptions requested in 1.24
+- name: "Add support for resizing PVs"
+  issue: 284
+  date_requested: 2022-02-03
+  date_reviewed: 2022-02-03
```

Comment:
Looking at the mailing list it was filed at 2022-02-03 03:35:21 UTC. So either the 3rd or the 4th depending on your timezone. We gave the approval on 2022-02-04 17:19:58 UTC, so I'd say that's the 4th for sure.